### PR TITLE
[test](ut) added UT cases for show build index

### PR DESCRIPTION
**Pull request by Nitin-Kashyap**
## Proposed changes
Added UT case for show build index flow

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowBuildIndexStmt.java [hunk 1]
```diff
@@ -136,7 +136,7 @@ public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
 
     private void analyzeSubPredicate(Expr subExpr) throws AnalysisException {
         if (subExpr == null) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowBuildIndexStmt.java:138]: I think we can simply remove the `if` in line 99.
And not change here.

Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowBuildIndexStmt.java:138]: BTW, could you please also modify the comment in line 44, that is a typo.
Thank you

Comment (Nitin-Kashyap) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowBuildIndexStmt.java:138]: fixed

**Comment by Nitin-Kashyap** [comment]
run buildall

**Comment by Nitin-Kashyap** [comment]
run buildall

**Review by Vallishp — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**